src/libs/ValidationUtils.js

```diff
@@ -138,6 +138,19 @@ function isValidDebitCard(string) {
     return validateCardNumber(string);
 }
 
+/**
+ *
+ * @param {String} nameOnCard
+ * @returns {Boolean}
+ */
+function isValidCardName(nameOnCard) {
+    if (!CONST.REGEX.ALPHABETIC_CHARS.test(nameOnCard)) {
```

Comment:
Is there any reason that we want to allow numbers at all? Is that a valid case?

Reply:
I'm not sure about this one. The [WAF in Web-Secure](https://github.com/Expensify/Web-Secure/blob/9be43a721ae07155b194fbbd26a7fe2a16853674/_inputrules.php#L81) allows entering any value and also the [regex in old dot](https://github.com/Expensify/Web-Secure/blob/d35db446442fdb41cff20604a9584a7810ea9c1f/site/app/dialogs/addBillingCard/addBillingCard.jsx#L111) checks if there's at least one alphabetic character. 

The comment in the WAF rule makes me think that maybe there are companies that use numbers or other characters in the card names (which could not be related to a specific person but a role or area)? 🤔

Reply:
I have [asked in Slack](https://expensify.slack.com/archives/C01GTK53T8Q/p1640177249460500) to see if anyone else knows for sure.

Reply:
Okay, apparently it is possible, so lets merge

Reply:
https://github.com/Expensify/App/pull/21880 - came as missing part - to prevent adding emojis inside card name